Workspace/src/ping_test/src/ping_server.cpp

```diff
@@ -0,0 +1,76 @@
+#include "ping_test.h"
```

Comment:
So this is more like a heartbeat/watchdog type thing?

Reply:
yeah, the idea was to test the connection between two nodes on the same network

Reply:
Ok sounds good